src/main/kotlin/racingcar/ui/ResultView.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.ui
+
+import racingcar.application.Car
+
+object ResultView {
+    fun print(cars: List<Car>) {
+        cars.forEach { car ->
+            println(makeLine(count = car.step))
+        }
+        println()
+    }
+
+    private fun makeLine(count: Int): String =
+        StringBuilder()
+            .append("-".repeat(count))
+            .toString()
```

Comment:
단순히 String의 `repeat` 확장함수만 호출해도 동일한 결과를 얻을 수 있을 것 같아요.  
이 로직은 makeLine이라는 메서드로 추출할 필요가 없을 것 같네요. 

```suggestion
    private fun makeLine(count: Int): String = "-".repeat(count)
```

Reply:
네! 반영하도록 할게요🙇‍♂️